src/pages/debrief/debrief.ts

```diff
@@ -239,13 +239,14 @@ export class DebriefPage extends PracticeableBasePageComponent {
       return;
     }
     this.navController.push(BACK_TO_OFFICE_PAGE).then(() => {
-      const testReportPage = this.navController.getViews().find(view => view.name === TEST_REPORT_PAGE);
+      const testReportPage = this.navController.getViews().find(view => view.id === TEST_REPORT_PAGE);
```

Comment:
Is there any documentation on what this `id` property actually relate to? I couldn't find anything on https://ionicframework.com/docs/v3/api/navigation/ViewController/

Reply:
It's undocumented, but seems to be the only way to find a view in the stack without resorting to counting back through indexes. If we don't use this, I think we need to abandon the idea of popping pages off and rely on `popToRoot` clearing up after the write up.

In the event that `id` is removed or changed (unlikely?) then the additional checking means that we simply wouldn't remove any pages.